covalent_dispatcher/_core/execution.py

```diff
@@ -611,6 +639,85 @@ async def _postprocess_workflow(result_object: Result) -> Result:
     return result_object
 
 
+async def _reconstruct_sublattice_result(result_object: Result) -> Result:
+    """
+    Reconstruct a sublattice workflow result by making electron output substitutions.
+
+    Args:
+        result_object: Result object being used for current dispatch
+
+    Returns:
+        The result object with a restructured result value
+    """
+    rc_executor = result_object.lattice.get_metadata("workflow_executor")
+    rc_executor_data = result_object.lattice.get_metadata("workflow_executor_data")
+    selected_executor = [rc_executor, rc_executor_data]
+
+    result_object._status = Result.PENDING_RECONSTRUCTION
+    upsert._lattice_data(result_object)
+
+    app_log.debug(f"Preparing to reconstruct sublattice result {result_object.dispatch_id}")
+
+    reconstruct_inputs = {
+        "args": [TransportableObject.make_transportable(result_object._lattice)],
```

Comment:
Hi @araghukas , can we avoid passing the entire lattice as input to the reconstructor? Sending the whole lattice would negate any memory savings since the lattice contains the entire transport graph together with all intermediate node output. Your reconstruction procedure only needs the following information:

* The return value of build graph with the placeholders
* A map of `node_id` to node output for the `node_ids` referenced in the return value (`return_info["electron_ids"]`)